ingestion-data/production/discovery-items/lis-global-da-gpp-trend.json

```diff
@@ -1,8 +1,28 @@
-{
-    "collection": "lis-global-da-gpp-trend",
-    "bucket": "veda-data-store",
-    "prefix": "lis-global-da-gpp-trend/",
-    "filename_regex": "^(.*)DAGPP_STL_based_trend",
-    "discovery": "s3",
-    "upload": false
-}
\ No newline at end of file
+[
+    {
+        "collection": "lis-global-da-gpp-trend",
+        "bucket": "veda-data-store",
+        "prefix": "lis-global-da-gpp-trend/",
+        "filename_regex": "^(.*)DAGPP_STL_based_trend_v3_2003_2021.cog.tif",
+        "discovery": "s3",
+        "upload": false
+    },
+    {
```

Comment:
I'm pretty sure we do not want to catch the v2 version so we probably want to remove this block of discovery input. 

It's easier to see here http://veda-staging-stac-browser.s3-website-us-west-2.amazonaws.com/collections/lis-global-da-gpp-trend

Reply:
I can't access that page, @anayeaye 😯 it says the key doesn't exist

Reply:
https://staging-stac.delta-backend.com/collections/lis-global-da-gpp-trend/items so even though staging has the v2 items, we don't want to propagate that?

Reply:
sorry, I forget that copying the browser url doesn't really work for sharing as expected :(

Let's leave it as is, I just opened an issue for us to come back to this if time permits.
https://github.com/NASA-IMPACT/veda-data/issues/131